doc/how_to/make_a_sorting.rst

```diff
@@ -0,0 +1,87 @@
+(e.g. correlograms or for generating a :code:`SortingAnalyzer`)
+
+
+Making a :code:`Sorting`
+------------------------
+
+For most formats the :code:`Sorting` is automatically generated. For example one could do
+
+.. code-block:: python
+
+    from spikeinterface.extractors import read_kilosort, read_phy
+
+    # For kilosort/phy files we can use either reader
+    ks_sorting = read_kilosort('path/to/folder')
+    phy_sorting = read_phy('path/to/folder')
+
+This :code:`Sorting` contains important information about your spike trains including
+the spike times (i.e. when the neurons were actually firing) the unit labels (i.e.
+who the spikes belong to. Also called cluster ids by some sorters), the unit ids (the unique
+set of unit labels) and the sampling_frequency. To make your own :code:`Sorting` object you can
+use :code:`NumpySorting`. It is important to note if you are new to spike trains that they
+are typically stored in samples/frames rather than in seconds. So you should input the times
+in samples/frames. The sampling_frequency allows for easily switching between samples and seconds.
+
+There are 3 options (along with making a NumpySorting from another sorting which will not be covered here):
+
+With lists of spike trains and spike labels
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this case we need a list or array (or lists of lists for multisegment) of spike times,
+unit labels, sampling_frequency and optional unit_ids if you want specific labels to be
+used (in this case we only create the :code:`Sorting` based on the requested unit_ids).
+
+.. code-block:: python
+
+    from spikeinterface.core import NumpySorting
+
+    # in this case we are making a monosegment sorting
+    my_sorting = NumpySorting.from_times_labels(times_list = [1,2,3,4],
+                                                labels_list = [0,1,0,1],
+                                                sampling_frequency = 30_000.0
+                                                )
+
+
+With a unit dictionary
+^^^^^^^^^^^^^^^^^^^^^^
+
+We can also use a dictionary where each unit is a key and its spike times are values.
+This is entered as either a list of dicts with each dict being a segment or as a single
+dict for monosegment. We still need to separately specify the sampling_frequency
+
+.. code-block:: python
+
+    from spikeinterface.core import NumpySorting
+
+    my_sorting = NumpySorting.from_unit_dict(units_dict_list={'0': [1,3],
+                                                              '1': [2,4]
+                                                              },
+                                            sampling_frequency=30_000.0
+                                           )
+
+
+With Neo SpikeTrains
+^^^^^^^^^^^^^^^^^^^^
+
+Finally since SpikeInterface is tightly integrated with the Neo project you can create
+a sorting from :code:`Neo.SpikeTrain` objects. See Neo documentation for more information on
```

Comment:
I forgot how to do the auto-link so if someone can just remind me in review we should link to the Neo docs.

Reply:
Works like:
Please read the :doc:`Neo documentation<neo:index>`.

(currently we've got neo and probeinterface registered to be used like this)